<?xml version="1.0"?>
<launch>

<!-- blue side robot -->
  <group ns="blue_bot">
      <!-- Your robot control node run -->
      <include file="$(find burger_war)/launch/your_burger_sim.launch">
        <arg name="robot_name" value="blue_bot" />
      </include>
  </group>

<!-- red side robot -->
  <!-- Your robot control node run -->
  <include file="$(find burger_war)/launch/your_burger.launch">
    <arg name="side" value="r" />
  </include>

</launch>

